pkg/kubelet/network/kubenet/kubenet_linux.go

```diff
@@ -467,6 +491,36 @@ func (plugin *kubenetNetworkPlugin) Status() error {
 	return nil
 }
 
+func (plugin *kubenetNetworkPlugin) getRunningPods() ([]*hostport.RunningPod, error) {
+	pods, err := plugin.host.GetRuntime().GetPods(false)
+	if err != nil {
+		return nil, fmt.Errorf("Failed to retrieve pods from runtime: %v", err)
+	}
+	runningPods := make([]*hostport.RunningPod, 0)
+	for _, p := range pods {
+		for _, c := range p.Containers {
+			if c.Name != dockertools.PodInfraContainerName {
```

Comment:
Add a comment here for something like "Assuming pod specs yield by runtime.GetPods include name and ID of containers in pod"